### PR TITLE
Fix pebble cache typo

### PR DESCRIPTION
**Pull request by maggie-lou**
**Related issues**: (https://github.com/buildbuddy-io/buildbuddy-internal/pull/2991)

### CODE REVIEW

**Comment by maggie-lou** [comment]
Current dependencies on/for this PR:
* `master`
  * **PR #5526** <a href="https://app.graphite.dev/github/pr/buildbuddy-io/buildbuddy/5526?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/buildbuddy-io/buildbuddy/5526?utm_source=stack-comment).

File: enterprise/server/backends/pebble_cache/pebble_cache.go [hunk 1]
```diff
@@ -69,7 +69,7 @@ var (
 	atimeBufferSizeFlag       = flag.Int("cache.pebble.atime_buffer_size", DefaultAtimeBufferSize, "Buffer up to this many atime updates in a channel before dropping atime updates")
 	sampleBufferSize          = flag.Int("cache.pebbles.sample_buffer_size", DefaultSampleBufferSize, "Buffer up to this many samples for eviction sampling")
 	deleteBufferSize          = flag.Int("cache.pebbles.delete_buffer_size", DefaultDeleteBufferSize, "Buffer up to this many samples for eviction eviction")
-	numDeleteWorkers          = flag.Int("cache.pebbles.num_delete_workers", DefaultNumDeleteWorkers, "Number of deletes in parallel")
+	numDeleteWorkers          = flag.Int("cache.pebble.num_delete_workers", DefaultNumDeleteWorkers, "Number of deletes in parallel")
```
Comment (tylerwilliams) [inline enterprise/server/backends/pebble_cache/pebble_cache.go:72]: can we get the ones above too?

**Review by luluz66 — COMMENTED** [review]
Since we are using these flags in production, do we need to add the new flags first; and then delete the old flags after we cleaned up the old flag usages?

**Comment by maggie-lou** [comment]
> Since we are using these flags in production, do we need to add the new flags first; and then delete the old flags after we cleaned up the old flag usages?

Sanity check here. Should be safe: https://buildbuddy-corp.slack.com/archives/C01D5GHRJ59/p1702574253229449?thread_ts=1702574156.096339&cid=C01D5GHRJ59

**Review by luluz66 — APPROVED** [review]

**Pull request merged**